CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.80.0] - 2025-08-04
```

Comment:
A comment related to documentation, in the `model_revisions.py` file, we could update the docstring with dependencies as well. 

<img width="859" height="529" alt="Screenshot 2025-08-05 at 13 47 01" src="https://github.com/user-attachments/assets/e2563b90-ffa1-4fa5-b16b-26dfe7b1aeeb" />